Flexbox1/style1.css

```diff
@@ -0,0 +1,91 @@
+body{
+    margin: 0;
+    border: 0;
+}
+
+.parent{
+    display: flex;
+    justify-content: flex-start;
+    flex-wrap: wrap;
+}
+
+.card1{
+    background-color: rgb(251, 251, 251);
+    border: 1px solid grey;
+    padding: 8px;
+    max-width: 200px;
+    height: 320px;
+    margin: 10px;
+    align-items: center;
+    border-radius: 5px;
+    background-image: url("https://i.ebayimg.com/thumbs/images/g/BfgAAOSwBG5mFsxC/s-l1200.jpg");
+    background-size: contain;
+    background-repeat: no-repeat;
+    box-shadow: 0 0px 16px 0 rgba(128, 128, 128, 0.3);
+}
+
+.card2{
+    background-color: rgb(251, 251, 251);
+    border: 1px solid grey;
+    padding: 8px;
+    max-width: 200px;
+    height: 320px;
+    margin: 10px;
+    align-items: center;
+    border-radius: 5px;
+    background-image: url("https://assets.bwbx.io/images/users/iqjWHBFdfxIU/i2nEtFZqb01I/v0/-1x-1.jpg");
+    background-size: contain;
+    background-repeat: no-repeat;
+    box-shadow: 0 0px 16px 0 rgba(128, 128, 128, 0.3);
+}
+
+.card3{
+    background-color: rgb(251, 251, 251);
+    border: 1px solid grey;
+    padding: 8px;
+    max-width: 200px;
+    height: 320px;
+    margin: 10px;
+    align-items: center;
+    border-radius: 5px;
+    background-image: url("https://robbreport.com/wp-content/uploads/2022/06/favtrio.gif");
+    background-size: contain;
+    background-repeat: no-repeat;
+    box-shadow: 0 0px 16px 0 rgba(128, 128, 128, 0.3);
+}
+
+.card4{
+    background-color: rgb(251, 251, 251);
+    border: 1px solid grey;
+    padding: 8px;
+    max-width: 200px;
+    height: 320px;
+    margin: 10px;
+    align-items: center;
+    border-radius: 5px;
+    background-image: url("https://media.istockphoto.com/id/1180244659/photo/luxury-watch-isolated-on-white-background-with-clipping-path-for-artwork-or-design-black.jpg?s=612x612&w=0&k=20&c=yeFNfkQmcVV9BTUlZO8vY_oLOQgDAt23LfCbF1e3fbI=");
+    background-size: contain;
+    background-repeat: no-repeat;
+    box-shadow: 0 0px 16px 0 rgba(128, 128, 128, 0.3);
+}
```

Comment:
Ca si sugestie, avem mult cod pentru stil care se repeta pentru carduri. Avem 2 optiuni:

1. Putem crea o clasa numita `card` care sa contina codul ce se repeta si cate 1 clasa pentru fiecare card care sa contina `background-mage`.
2. In loc sa ne folosim de `background-image` direct pe div, putem sa cream copii in interorul div-urilor card cu tag-uri de `<img src="link-catre-imagine" />` si pastram o singura clasa pt `card`

```suggestion
.card {
  background-color: rgb(251, 251, 251);
  border: 1px solid grey;
  padding: 8px;
  max-width: 200px;
  height: 320px;
  margin: 10px;
  align-items: center;
  border-radius: 5px;
  background-size: contain;
  background-repeat: no-repeat;
  box-shadow: 0 0px 16px 0 rgba(128, 128, 128, 0.3);
}

.card1 {
  background-image: url("https://i.ebayimg.com/thumbs/images/g/BfgAAOSwBG5mFsxC/s-l1200.jpg");
}
.card2 {
  background-image: url("https://assets.bwbx.io/images/users/iqjWHBFdfxIU/i2nEtFZqb01I/v0/-1x-1.jpg");
}
.card3 {
  background-image: url("https://robbreport.com/wp-content/uploads/2022/06/favtrio.gif");
}
.card4 {
  background-image: url("https://media.istockphoto.com/id/1180244659/photo/luxury-watch-isolated-on-white-background-with-clipping-path-for-artwork-or-design-black.jpg?s=612x612&w=0&k=20&c=yeFNfkQmcVV9BTUlZO8vY_oLOQgDAt23LfCbF1e3fbI=");
}
```